lib/internal/modules/esm/default_resolve.js

```diff
@@ -6,8 +6,9 @@ const internalFS = require('internal/fs/utils');
 const { NativeModule } = require('internal/bootstrap/loaders');
 const { extname } = require('path');
 const { realpathSync } = require('fs');
-const preserveSymlinks = !!process.binding('config').preserveSymlinks;
-const preserveSymlinksMain = !!process.binding('config').preserveSymlinksMain;
+const { getOptions } = internalBinding('options');
+const preserveSymlinks = !!getOptions('--preserve-symlinks');
```

Comment:
please remove the `!!`, `getOptions` already returns true/false

Reply:
Should be removed now!